api/scripts/update-certification-challenges-with-skill-id.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const _ = require('lodash');
+const Promise = require('bluebird');
+const { knex } = require('../lib/infrastructure/bookshelf');
+const skillDatasource = require('../lib/infrastructure/datasources/airtable/skill-datasource');
+
+async function getDistinctAssociatedSkillNamesForNullSkillId() {
+  const result = await knex.raw('SELECT DISTINCT ?? FROM ?? WHERE ?? IS NULL', ['associatedSkillName', 'certification-challenges', 'associatedSkillId']);
+  return _.map(result.rows, 'associatedSkillName');
+}
+
+async function mapSkillNamesWithId(skillNames) {
+  return Promise.map(skillNames, async (skillName) => {
+    const skills = await skillDatasource.findByName(skillName);
+    if (_.isEmpty(skills)) {
+      return {
+        skillName,
+        skillId: null,
+      };
+    }
+    return {
+      skillName,
+      skillId: skills[0].id,
+    };
```

Comment:
Je fais une petite suggestion parce que je trouve la ternaire simple en l'occurence mais je pourrais comprendre que ça ne convienne pas à tout le monde : 
```suggestion
    return {
      skillName,
      skillId: _.isEmpty(skills) ? null : skills[0].id,
    };
```

Reply:
Ah oui bien vu !merci :+1: